browse-everything.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'google-api-client', '~> 0.9'
   spec.add_dependency 'signet'
   spec.add_dependency 'httparty'
-  spec.add_dependency 'aws-sdk'
+  spec.add_dependency 'aws-sdk-s3'
```

Comment:
I think you want a "~> 1.0" version specification on this dep, no?  They started aws-sdk-s3 at 1.0. 2.0 would presumably be a future version with backwards breaking changes.